test/test.idl

```diff
@@ -0,0 +1,81 @@
+#include <dds/DdsDcpsCore.idl>
+
+module test {
+
+struct BasicTypes {
+  unsigned short us;
+  short s;
+  unsigned long ul;
+  long l;
+  unsigned long long ull;
+  long long ll;
```

Comment:
What about `uint8` and `int8`?

Reply:
I'm not sure OpenDynamicData has support for those, and we're covering octet and char already.